File: examples/ask-anything.py
"""Ask any financial question — get structured intelligence."""
from veroq import VeroqClient

client = VeroqClient()
result = client.ask("How is NVDA doing?")

print(result["summary"])
print(f"Confidence: {result['confidence']['level']}")
print(f"Trade signal: {result['trade_signal']['action']} ({result['trade_signal']['score']}/100)")
print(f"Endpoints called: {result['endpoints_called']}")

File: examples/ask-anything.ts
import { VeroqClient } from '@veroq/sdk';

const client = new VeroqClient();
const result = await client.ask("How is NVDA doing?");

console.log(result.summary);
console.log(`Trade signal: ${result.trade_signal?.action} (${result.trade_signal?.score}/100)`);
